client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionEventForm/PlacesAdditionalForms/TransportationAdditionalForms/TrainEventForm.tsx

```diff
@@ -58,11 +63,16 @@ const TrainEventForm : React.FC = () : JSX.Element => {
                     <FormInput fieldName='עיר יעד'>
                         <Autocomplete
                             options={Array.from(cities, ([id, value]) => ({ id, value }))}
-                            getOptionLabel={(option) => option.value.displayName}
-                            inputValue={cities.get(cityDestination as string)?.displayName}
+                            getOptionLabel={(option) => option.value?.displayName || ''}
+                            defaultValue={{ id: cityDestination as string, value: cities.get(cityDestination as string)}}
                             onChange={(event, selectedCity) => {
                                 onChange(selectedCity?.id as string, InteractionEventDialogFields.CITY_DESTINATION)
                             }}
+                            onInputChange={(event, newInputValue) => {
```

Comment:
what is the difference between onInputChange and onChange?

Reply:
onInputChange == deleting with backspace
onChange == choosing from the options

Reply:
וזה גם מכסה מקרה של בחירה?

Reply:
בחירה זה onChange, מה שקורה בonInputChange אמור להיות מכוסה בוולידציות לדעתי